Userland/Libraries/LibWeb/HTML/HTMLCanvasElement.cpp

```diff
@@ -35,40 +35,100 @@ unsigned HTMLCanvasElement::height() const
     return attribute(HTML::AttributeNames::height).to_uint().value_or(150);
 }
 
+void HTMLCanvasElement::reset_context_to_default_state()
+{
+    m_context.visit(
+        [](NonnullRefPtr<CanvasRenderingContext2D>& two_dimensional_context) {
+            two_dimensional_context->reset_to_default_state();
```

Comment:
Just `context` for the variable name is fine IMO, `two_dimensional` seems a bit awkward